### PR TITLE
chore: Verbose release notes

### PR DESCRIPTION
**Pull request by crenshaw-dev**
I'd hoped we could use auto-generated release notes, but that wasn't working (particularly with big releases).

### CODE REVIEW

File: hack/generate-release-notes.sh [hunk 1]
```diff
@@ -103,3 +103,67 @@ if [ "$new_contributors_num" -lt 20 ] && [ "$new_contributors_num" -gt 0 ]; then
...
+
+echo "## Changes"
+echo
+echo "This release includes $new_commits_num contributions from $contributors_num contributors$new_contributors_message with $features_num features and $fixes_num bug fixes."
+echo
+if [ "$new_contributors_num" -lt 20 ] && [ "$new_contributors_num" -gt 0 ]; then
+  echo "A special thanks goes to the $new_contributors_num new contributors:"
+  echo "$new_contributors_names"
+  echo
+fi
```
Comment (34fathombelow) [inline hack/generate-release-notes.sh:135]: Should lines 72-105 be truncated?

File: hack/generate-release-notes.sh [hunk 2]
```diff
@@ -103,3 +109,27 @@ if [ "$new_contributors_num" -lt 20 ] && [ "$new_contributors_num" -gt 0 ]; then
   echo "$new_contributors_names"
   echo
 fi
+if [ "$features_num" -gt 0 ]; then
+  echo "### Features"
```
Comment (leoluz) [inline hack/generate-release-notes.sh:113]: How about adding the `features_num` right next to the title?

File: hack/generate-release-notes.sh [hunk 3]
```diff
@@ -76,12 +76,18 @@ more_log=$(git log --pretty="format:%s %ae" "$new_ref..$old_ref")
 
 new_commits=$(diff --new-line-format="" --unchanged-line-format="" <(echo "$less_log") <(echo "$more_log") | grep -v "Merge pull request from GHSA")
 new_commits_no_email=$(echo "$new_commits" | strip_last_word)
+features=$(echo "$new_commits_no_email" | grep '^feat' | to_list_items)
```
[the remark this answers is not in the document]
Comment (crenshaw-dev) [reply hack/generate-release-notes.sh:79]: Sometimes folks do `feat(something)`. And occasionally the infuriating `feature:`. :-P

**Comment by crenshaw-dev** [comment]
@leoluz PTAL

**Review by leoluz — APPROVED** [review]
LGTM

**Pull request merged**